third_party/nvidia/hopper/lib/Transforms/WarpSpecialization/WSLowerMem.cpp

```diff
@@ -355,6 +355,7 @@ Operation *optimizeTMALoads(OpBuilderWithAsyncTaskIds &builder,
     copy = builder.createWithAsyncTaskIds<ttng::AsyncTMACopyGlobalToLocalOp>(
         loc, tmaLoad.getDesc(), tmaLoad.getIndices(), prodBarrier,
         pipelineBuffer, pred);
+    copyLoopScheduleInfo(copy, tmaLoad);
```

Comment:
This doesn't change the IR yet because we don't have SWP in the producer Partition, but if that partition had multiple stages it would be wrong without this change.